src/store/index.js

```diff
@@ -93,6 +97,7 @@ export default createStore({
           { classname: "BardBot", selected: true },
           { classname: "Llama270bBot", selected: true },
           { classname: "VicunaBot", selected: true },
+          { classname: "ChatGLMBot", selected: true },
```

Comment:
```suggestion
          { classname: "ChatGLMBot", selected: true },
```
Only recommended or no-login-required bots will be added here.